backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -226,38 +230,6 @@ public void init() { // setup default stuff
     }
   }
 
-  private void initOidc() {
-    // oidc settings takes priority over env variables
-    String isOidcEnabledSetting = getSetting(Constants.IS_OIDC_ENABLED);
-    if (isOidcEnabledSetting != null) {
-      this.isOidcEnabled = Boolean.parseBoolean(isOidcEnabledSetting);
-    } else {
-      Object envSetting =
-          EnvironmentProperty.getParameter(Constants.MOLGENIS_OIDC_CLIENT_ID, null, STRING);
```

Comment:
I think we should at some point have some way to find out about oidc errors but this is probably not the way.